proposals/mdawson-cpc-charter/CPC-Charter.md

```diff
@@ -173,28 +121,44 @@ ensuring:
      community.
    * defining and maintaining neutral consensus for the technical vision for the _$FOUNDATION_
-   * approving new projects within the scope for _$FOUNDATION_ set by the Governing Board,
-     as outlined in [XXXX], and creating conceptual architecture for how the proejcts
-     fit together.
-   * aligning projects, removing or archiving projects, as outlined in [XXXX].
-   * approving funds for budgets delegated to the project. A good example is the
-     travel fund.
+   * creating conceptual architecture for how the proejcts fit together.
+   * Voting members are responsible for approving new projects within the scope
+     for _$FOUNDATION_ set by the Governing Board, as outlined in [XXXX].
+   * Voting mebmers are responsible for making final decisions on aligning projects,
+     removing or archiving projects, as outlined in [XXXX].
+   * Voting members are responsible for approving funds for budgets delegated 
+     to the project. A good example is the travel fund.
    * follow and be up to date on Board/$FOUNDATION initiatives and communicate them to the projects.
    * defining common practices to be implemented across _$FOUNDATION_ projects, if any.
    * The CPC is also responsible for mediating technical conflicts between
      _$FOUNDATION_ Projects when attempts to resolve those conflicts within the Project
    were unsuccessful. 
+   * Creating a yearly budget that can be drawn on by the projects.
    * The CPC serves as the _$FOUNDATION_'s primary technical
      liaison body with external open source projects, consortiums and groups and is
      also responsible for ensuring that the _$FOUNDATION_ has appropriate technical
      participation in standards bodies by finding an encouraging projects members to
      particiapte in these bodies.
+   * Managing the Individual membership program
+   * Managing programs to improve Inclusivity and Diversity
+   * Guiding projects into appropriate Foundation paths
+   * Shared resources, tooling and funding for projects and Commons.js (Events
+     Collabl Summit, Travel fund)
+   * Code of Conduct Indident response support
+   * Developing a framework for community end-user feedback
+   * Managing programs to offer mentorship to external individuals
+   * Managing developer outreach programs
+   * Support project advocacy and regonition programs
+   * Identify, recruit and engage prospective projects
+   * Voting members are responsible for making a final decision if consensus
+     cannot be reached among members.
    * Members of the CPC, or committees consisting of subsets of the CPC
      membership will be expected to meet on a regular basis to discuss topics
      such as new project acceptance into the mentorship program, project graduation
      from the mentorship program, Project issues and conflicts, opportunities
      for collaboration between Projects, opportunities for the Foundation in the
      greater JavaScript community, etc.
```

Comment:
I would expect the CPC to create subcommittees to delegate/decide/make proposals on significant topics. Can we allow this more explicitly?

Reply:
Added